layouts/shortcodes/iframe.html

```diff
@@ -0,0 +1,5 @@
+{{ $url := .Get "url" }}
+<div style="width: 100%; height:calc(100vh - 200px)">
+    <iframe src="{{ $url }}" height="100%" width="100%"><a href="{{ $url }}">API Documentation</a></iframe>
```

Comment:
There's a leaky abstraction here. This shortcode looks generic, yet it has the text "API Documentation" in a link beneath the iframe. What does this link do? Is it necessary?